nifi-nar-bundles/nifi-kudu-bundle/nifi-kudu-processors/src/test/java/org/apache/nifi/processors/kudu/ITPutKudu.java

```diff
@@ -139,6 +139,12 @@ public void testWriteKudu() throws IOException, InitializationException {
         testRunner.setProperty(PutKudu.BATCH_SIZE, "10");
         testRunner.setProperty(PutKudu.FLOWFILE_BATCH_SIZE, "2");
 
+        // Enable lowercase handling.
+        testRunner.setProperty(PutKudu.LOWERCASE_FIELD_NAMES, "true");
+
+        // Enable schema drift handling.
+        testRunner.setProperty(PutKudu.HANDLE_SCHEMA_DRIFT, "true");
```

Comment:
Could be worth doing the test using flow file attributes. Just to check EL evaluation.